vite.config.js

```
@@ -1,24 +1,22 @@
+import { defineConfig, loadEnv } from 'vite';
 import react from '@vitejs/plugin-react';
 import tsconfigPaths from 'vite-tsconfig-paths';
 import tailwindcss from '@tailwindcss/vite';
-import { API_BASE_URL } from '../config/env';
 
-export default defineConfig({
-  plugins: [
-    react(),
-    tailwindcss(),
+export default defineConfig(({ mode }) => {
+  const env = loadEnv(mode, process.cwd());
```

ESLint

[error] 'process' is not defined.

(no-undef)